src/deriver/ScopedKeys.js

```diff
@@ -63,6 +66,11 @@ class ScopedKeys {
         throw new Error('keyRotationTimestamp must be a 13-digit number');
       }
 
+      if (options.identifier === LEGACY_SYNC_SCOPE) {
```

Comment:
should we update the `options.identifier ` doc string above about the new possibilities?

Reply:
I think the existing description actually covers it, it is still "a unique URI string identifying the requested scoped key".  But I'll try to add some more notes in the docstrings about this special behaviour.

Reply:
👍  yea i think i was leaning towards https://github.com/mozilla/fxa-crypto-relier/pull/14#discussion_r182293091 more in here